app/src/main/res/layout/event_detailbar.xml

```diff
@@ -9,26 +9,54 @@
     android:paddingBottom="4dp">
 
     <TextView
-        android:id="@+id/lecture_detailbar_date_time_location"
+        android:id="@+id/lecture_detailbar_date_time"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="8dp"
         android:layout_marginLeft="8dp"
         android:layout_marginEnd="8dp"
         android:layout_marginRight="8dp"
+        android:drawableStart="@drawable/ic_access_time"
+        android:drawableLeft="@drawable/ic_access_time"
         android:ellipsize="end"
         android:fontFamily="sans-serif-light"
+        android:gravity="center"
         android:maxLines="1"
         android:textAllCaps="true"
         android:textColor="@color/lecture_detailbar_text"
         android:textSize="@dimen/event_detailbar_text"
         android:typeface="normal"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toStartOf="@+id/lecture_detailbar_lecture_id"
         app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintStart_toEndOf="@id/lecture_detailbar_location"
         app:layout_constraintTop_toTopOf="parent"
         tools:text="@string/placeholder_event_date" />
 
+    <TextView
+        android:id="@+id/lecture_detailbar_location"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
+        android:drawableStart="@drawable/ic_room"
+        android:drawableLeft="@drawable/ic_room"
+        android:drawablePadding="@dimen/event_detailbar_icon_padding"
+        android:ellipsize="end"
+        android:fontFamily="sans-serif-light"
+        android:gravity="center_horizontal|left|center_vertical"
```

Comment:
Can you please use `start` instead of `left`? Lint warning about this.

Reply:
i did not know about start. I did not get any lint warning in the android studio. I read about it after seeing your suggestion - https://developer.android.com/reference/android/view/Gravity.html#START I add them in the rebase commit 
c435cfe